deploy/scripts/src/mobile/utils/README

```diff
@@ -1 +1,38 @@
-url.js from https://jslib.k6.io/url/1.0.0/index.js
\ No newline at end of file
+# Mobile Performance Testing
+
+This repository contains the performance test framework for testing DCMAW journeys.
+
+
+## Required Installations
+
+Pre-commit for running pre-commit hooks locally
+
+```bash
+brew install pre-commit && pre-commit install && pre-commit install -tprepare-commit-msg -tcommit-msg
+```
+
+k6 for testing and executing scripts locally
+```bash
+brew install k6
+```
+
+Install test dependencies
+```bash
+npm install
+```
+
+
+## Run Command
+K6 environment variales can be found in test-mobile-journey.ts file.
+Profile and scenario can be found in test.ts file.
+
+
+```bash
+# Generating dist files
+npm start
+```
+
+```bash
+# Run k6 test. Replace with actual values
+k6 run ./dist/mobile/test.js -e PROFILE=profile_name -e SCENARIO=scenario_name -e MOBILE_TEST_CLIENT_EXECUTE_URL=test_client_execute_url -e MOBILE_BACK_END_URL=backend_url -e MOBILE_FRONT_END_URL=frontend_url -e MOBILE_BIOMETRIC_SESSION_ID=biometric_session_id
```

Comment:
Tiny thing but I think it's easier for new joiners if you make the command look more like:

```
-e MOBILE_BIOMETRIC_SESSION_ID=<your_biometric_session_id>
```